### PR TITLE
Fix casing of initial `sv.` in celebration names

### PR DESCRIPTION
**Pull request by tukusejssirs**

### CODE REVIEW

File: data/czech-cs.txt [hunk 1]
```diff
@@ -263,4 +263,4 @@ country: cz
 27 f john_evangelist : Sv. Jana, apoštola a evangelisty
 28 f R innocents : Svatých Mláďátek, mučedníků
 29 R becket : Sv. Tomáše Becketa, biskupa a mučedníka
-31 sylvester_i : Sv. Silvestra I. papeže
```
Comment (igneus) [inline data/czech-cs.txt:266]: Looks like you (or perhaps your editor automatically) removed the newline at the end of the last line. Please remove this change, we do want the newline there.

**Comment by igneus** [comment]
Thank you. As I said before, I love pedantic proof-readers. I haven't ever (and quite probably I *wouldn't* ever) notice the lowercase `s`s.

File: data/czech-cs.txt (hunk 1 above)
Comment (tukusejssirs) [reply data/czech-cs.txt]: Oh, sorry. Yes, I have configured my global `.editorconfig` to remove trailing lines and I didn’t check the changes I was about to commit. My bad.

**Comment by igneus** [comment]
Thanks again, merging.

**Pull request merged**